### ISSUE TITLE
Loss does not decline

### ISSUE THREAD

**Issue by yooyo-Q** [qqwweee/keras-yolo3#325]
Regardless of how I change the number of iterations and learning rate, loss can only be reduced to 200-300. What is the solution?(无论我如何改迭代次数和学习率，loss只能降低到200-300之间，请问下有什么解决办法么)

**Comment by RobinDong** [comment]
是用的 Adam 优化器吗？
试试 Adam(lr=1e-4, amsgrad=True)

**Comment by eain3314** [comment]
你好 我改完之后loss还是50多和之前一样

**Comment by eain3314** [comment]
请问还有可能是哪里的原因 谢谢
